go/chat/attachments/preprocess.go

```diff
@@ -3,6 +3,8 @@ package attachments
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/keybase/client/go/chat/globals"
```

Comment:
Nit: could you use `goimports` or some other such import adder so these imports end up in the same places as other files?